etc/config/zig.amazon.properties

```diff
@@ -0,0 +1,15 @@
+compilers=&zig
+defaultCompiler=trunk
+
+group.zig.compilers=trunk
+group.zig.objdumper=/opt/compiler-explorer/gcc-8.1.0/bin/objdump
+group.zig.isSemVer=true
+group.zig.baseName=zig
+
+compiler.trunk.exe=/opt/compiler-explorer/zig-master/bin/zig
```

Comment:
When testing locally, this path does not seem to be correct.
Running the insallation process creates: 
```
~ » j explorer
/opt/compiler-explorer
/opt/compiler-explorer » cd zig-master 
/opt/compiler-explorer/zig-master » ls -l
total 78532
drwxr-xr-x 3 root root     4096 sep 21 17:08 lib
-rwxr-xr-x 1 root root 80407968 sep 21 17:22 zig
```
The bin folder seems to be missing, and instead the zig executable is directly at the top level.